core-lib/Kernel.ns

```diff
@@ -630,18 +630,21 @@ class Kernel vmMirror: vmMirror = Object <: Value (
   protected class Block2 = Block ()
   (
     (* Evaluating *)
-    public value           = ( self value: nil )
     public value: argument = ( ^ vmMirror blockValue: self with: argument )
   )
 
   protected class Block3 = Block ()
   (
     (* Evaluating *)
-    public value                  = ( self value: nil with: nil )
-    public value: arg             = ( self value: arg with: nil )
     public value: arg1 with: arg2 = ( ^ vmMirror blockValue: self with: arg1 with: arg2 )
   )
 
+  protected class BlockArbitrary = Block ()
```

Comment:
Do we at this point still need the other block classes?
Couldn't we just do away with Block1-3?

Reply:
@richard-roberts any opinions/requirements?
How does Grace handle block activations when the number of arguments doesn't match?

Reply:
We just throw an error; from the Grace spec: "Requesting the apply(...) method evaluates the block; it is an error to provide the wrong number of arguments."